src/Uno.UI/ts/WindowManager.ts

```diff
@@ -66,7 +66,7 @@ namespace Uno.UI {
 					bootstrapperLoader.classList.add(WindowManager.unoKeepLoaderClassName);
 
 					// Skip creating local splash screen.
```

Comment:
Is this comment still valid?

Reply:
Yes, still is, but I fail a TypeScript so didn't realize resolve does not return 😁 ! So, another adjustment incoming

Reply:
Now should be correct